FirebaseAuth/Sources/Swift/Auth/Auth.swift

```diff
@@ -1240,7 +1273,18 @@ extension Auth: AuthInterop {
         actionCodeSettings: actionCodeSettings,
         requestConfiguration: self.requestConfiguration
       )
-      self.wrapAsyncRPCTask(request, completion)
+      #if os(iOS)
+        self.wrapInjectRecaptcha(request: request,
+                                 action: AuthRecaptchaAction.signInWithPassword) { result, error in
```

Comment:
Why AuthRecaptchaAction.signInWithPassword is used for a password reset?

Reply:
Great catch! Thanks! Fixed.